apps/meteor/app/e2e/client/rocketchat.e2e.room.ts

```diff
@@ -622,7 +622,7 @@ export class E2ERoom extends Emitter {
 	async encryptMessageContent(
 		contentToBeEncrypted: Pick<IMessage, 'attachments' | 'files' | 'file'> & Optional<Pick<IMessage, 'msg'>, 'msg'>,
 	) {
-		const data = new TextEncoder().encode(EJSON.stringify(contentToBeEncrypted));
+		const data = new TextEncoder().encode(EJSON.stringify(contentToBeEncrypted)) as Uint8Array<ArrayBuffer>;
```

Comment:
This assertion feels expendable, as the definition from lib "dom" ensures the return type as `Uint8Array<ArrayBuffer>`.